src/components/cc-badge/cc-badge.js

```diff
@@ -36,35 +39,67 @@ export class CcBadge extends LitElement {
     /** @type {BadgeIntent} Sets the accent color used for the badge. */
     this.intent = 'neutral';
 
+    /** @type {boolean} Whether the component should be displayed as skeleton. */
+    this.skeleton = false;
+
     /** @type {BadgeWeight} Sets the style of the badge depending on how much one wants it to stand out. */
     this.weight = 'dimmed';
   }
 
   render () {
+    const modes = {
+      dimmed: this.weight == null || this.weight === 'dimmed',
```

Comment:
Thanks !

I'm just wondering where our limit is on this and how I should handle this type of cases in the future.
What I mean is that when we do this, it means we support `null` as a value for `weight` but it's not documented anywhere because `weight` is supposed to only accept strings with specific values.
Also, we handle `null` but we don't handle an empty string for instance.

I'm ok with this though, It does not cause any issue so why not, this was more of a question than a request for change :wink: